docs/source/examples/array-numpy.rst

```diff
@@ -0,0 +1,13 @@
+Creating Dask arrays from NumPy arrays
+======================================
+
+We can create Dask arrays from NumPy arrays using ``da.from_array``.
+
+.. code-block:: python
+
+    >>> import numpy as np
+    >>> import dask.array as da
+    >>> x = np.arange(1000)
+    >>> y = da.from_array(x, chunks=(100))
+    >>> y.mean().compute()
+    499.5
```

Comment:
I added some trivial examples to get started here (since the only existing array example was the pickle files. Happy to change this to a more realistic example with NumPy or remove it for now.